modules/developer-cli-odo-adding-storage-to-the-application-components.adoc

```diff
@@ -5,15 +5,67 @@
 [id="adding-storage-to-the-application-components_{context}"]
 = Adding storage to the application components
 
-Persistent storage keeps data available between restarts of {odo-title}. You can add storage to your components with the `odo storage` command.
+Persistent storage keeps data available between restarts of {odo-title}.
+Use `odo storage` to add data that must persist, such database files, dependencies, build artifacts (for example, a `.m2` directory for Maven applications), to your application.
```

Comment:
This sentence needs some work and is missing an "as". Suggest something like:
Use the `odo storage` command to add persistent data to your application. Examples of data that must persist include database files, dependencies, and build artifacts, such as a Maven directory.

Reply:
@bobfuru thank you! I've been struggling for a while with this sentence. Changed.